media-libs/lv2/lv2-1.14.0.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="An extensible audio plugin interface"
+DESCRIPTION="LV2 is an open standard for audio plugins, used by hundreds \
+of plugins and other projects. At its core, LV2 is a simple stable \
+interface, accompanied by extensions which add functionality to support \
+the needs of increasingly powerful audio software."
+HOMEPAGE="http://lv2plug.in"
+COPYRIGHT="2006-2012 Steve Harris, David Robillard"
+LICENSE="Public Domain"
```

Comment:
They appear to be using a custom [license](http://lv2plug.in/git/cgit.cgi/lv2.git/tree/COPYING?h=v1.14.0) instead.

Reply:
How does one add this? Do we need to copy the COPYRIGHT file into licenses?